dags/vz_afd_ems_import.py

```diff
@@ -1,58 +1,97 @@
+        "opitem": "Vision Zero ETLs",
+        "opfield": f"{secrets_env_prefix}.BUCKET_ENV",
+    },
+    "AWS_ACCESS_KEY_ID": {
+        "opitem": "Vision Zero ETLs",
+        "opfield": f"common.AWS_ACCESS_KEY_ID",
+    },
+    "AWS_SECRET_ACCESS_KEY": {
+        "opitem": "Vision Zero ETLs",
+        "opfield": f"common.AWS_SECRET_ACCESS_KEY",
+    },
+    "BUCKET_NAME": {
+        "opitem": "Vision Zero ETLs",
+        "opfield": f"common.BUCKET_NAME",
+    },
+    "HASURA_GRAPHQL_ENDPOINT": {
+        "opitem": "Vision Zero ETLs",
+        "opfield": f"{secrets_env_prefix}.HASURA_GRAPHQL_ENDPOINT",
+    },
+    "HASURA_GRAPHQL_ADMIN_SECRET": {
+        "opitem": "Vision Zero ETLs",
+        "opfield": f"{secrets_env_prefix}.HASURA_GRAPHQL_ADMIN_SECRET",
+    },
+}
 
 
-# EMS
 @dag(
-    dag_id="vz-ems-import",
-    description="A DAG which imports EMS data into the Vision Zero database.",
-    schedule="0 7 * * *",
+    dag_id="vz-afd-ems-incident-import",
+    description="A DAG which imports EMS and AFD data into the Vision Zero database.",
+    schedule="0 7 * * *" if DEPLOYMENT_ENVIRONMENT == "production" else None,
     start_date=datetime(2023, 1, 1, tz="America/Chicago"),
     catchup=False,
-    tags=["repo:atd-vz-data", "vision-zero", "ems", "import"],
+    tags=["repo:atd-vz-data", "vision-zero", "ems", "afd", "import"],
     on_failure_callback=task_fail_slack_alert,
 )
-def etl_ems_import():
-    DockerOperator(
+def etl_data_import():
+    dag.byline = f"Failure impacts VZ team, {slack_member_ids['John']} & {slack_member_ids['Frank']}"
+    env_vars = get_env_vars_task(REQUIRED_SECRETS)
+
+    # EMS
+    ems_import = DockerOperator(
         task_id="run_ems_import",
-        environment=dict(os.environ),
-        image="atddocker/vz-afd-ems-import:production",
+        environment=env_vars,
+        image=docker_image,
         auto_remove="force",
-        entrypoint=["/entrypoint.sh"],
-        command=["ems"],
+        command="ems --skip-archive",
         tty=True,
         force_pull=True,
+        mount_tmp_dir=False
     )
 
-
-etl_ems_import()
-
-
-# AFD
-@dag(
-    dag_id="vz-afd-import",
-    description="A DAG which imports AFD data into the Vision Zero database.",
-    schedule="0 7 * * *",
-    start_date=datetime(2023, 1, 1, tz="America/Chicago"),
-    catchup=False,
-    tags=["repo:atd-vz-data", "vision-zero", "afd", "import"],
-    on_failure_callback=task_fail_slack_alert,
-)
-def etl_afd_import():
-    DockerOperator(
-        task_id="run_ems_import",
-        environment=dict(os.environ),
-        image="atddocker/vz-afd-ems-import:production",
+    # AFD
+    afd_import = DockerOperator(
+        task_id="run_afd_import",
+        environment=env_vars,
+        image=docker_image,
         docker_conn_id="docker_default",
         auto_remove="force",
-        entrypoint=["/entrypoint.sh"],
-        command=["afd"],
+        command="afd --skip-archive",
         tty=True,
         force_pull=True,
```

Comment:
does `force_pull=True` here mean that its going to pull the docker image again before running this task?

Reply:
it means that it's going to try to pull a newer image if it exists. but this is not needed on both tasks, so i will remove from this one 🙏